src/utilities/extendSchema.js

```diff
@@ -379,16 +399,18 @@ export function extendSchema(
     });
   }
 
-  function extendArgsMap(
-    args: GraphQLArgument[],
-  ): GraphQLArgument[] {
-    return args.map(arg => ({
-      name: arg.name,
-      type: extendFieldType(arg.type),
-      defaultValue: arg.defaultValue,
-      description: arg.description,
-      astNode: arg.astNode,
-    }));
+  function extendArgs(args: GraphQLArgument[]): GraphQLFieldConfigArgumentMap {
+    return keyValMap(
+      args,
+      arg => arg.name,
+      arg => ({
+        name: arg.name,
```

Comment:
You don't need `name`, see:
https://github.com/graphql/graphql-js/blob/ae5b163d2e6c124107fa0971f6d838c8a7d29f51/src/type/definition.js#L842-L847